src/physics.js

```diff
@@ -0,0 +1,58 @@
+import Vec2 from 'vec2';
+
+export default class Physics {
+    constructor(cameraBody, distanceFactor = 1.5, timeStep = 10, gravity = 9.82) {
+        this.cameraBody = cameraBody;
+        this.physicsBodyArr = new Array();
+        this.distanceFactor = distanceFactor;
+        this.timeStep = timeStep;
+        this.gravity = gravity; //units/(s^2)
+    }
+
+    addBody(physicsBody) {
+        this.physicsBodyArr.push(physicsBody);
+        this.cameraSort();
+    }
+
+    resolveView() {
```

Comment:
So instead of `resolveView` perhaps we should do `resolveArea(physicsBody)` so that this is more generic.  

So I'm imagining:

```
var objectsInArea = physicsEngine.resolveArea(viewportPhysicsBody);
```

Just increases re-usability if we need that for later for some reason.  Also separates the dependency on the camera/viewport to work.